src/Webform.js

```diff
@@ -862,9 +862,39 @@ export default class Webform extends NestedComponent {
     this.on('deleteSubmission', () => this.deleteSubmission(), true);
     this.on('refreshData', () => this.updateValue(), true);
 
+    this.executeFormController();
+
     return this.formReady;
   }
 
+  /**
+   * Performs an evaluation using the evaluation context of this component.
+   *
+   * @param func
+   * @param args
+   * @param ret
+   * @param tokenize
+   * @return {*}
+   */
+  evaluate(func, args, ret, tokenize) {
```

Comment:
Why do you need to rewrite this method?... Webform derives from Element which has this same method signature.